ugvc/pipelines/vcfeval_flavors.py

```diff
@@ -24,25 +24,36 @@
 
 from simppl.simple_pipeline import SimplePipeline
 from ugvc.utils.stats_utils import get_precision, get_recall, get_f1
+from ugvc.vcfbed.interval_file import IntervalFile
 
 
 def get_parser():
     ap = argparse.ArgumentParser(prog='vcfeval_flavors.py', description=run.__doc__)
     ap.add_argument('-b', '--baseline',
-                    help="VCF file containing baseline variants", type=str, required=True)
+                    help="VCF file containing baseline variants",
+                    type=str, required=True)
     ap.add_argument('-c', '--calls',
-                    help="VCF file containing called variants--output_prefix", type=str, required=True)
+                    help="VCF file containing called variants--output_prefix",
+                    type=str, required=True)
     ap.add_argument('-e', '--evaluation_regions',
-                    help='if set, evaluate within regions contained in the supplied BED file, '
-                         'allowing transborder matches. To be used for truth-set', type=str)
+                    help='if set, evaluate within regions contained in the intersection of the supplied bed files,'
+                         ' allowing transborder matches. To be used for truth-set',
```

Comment:
What does it mean that you allow transborder matches?

Reply:
I just coped the vcfeval help message...